backend/app/views/spree/admin/menus/index.html.erb

```diff
@@ -24,8 +24,8 @@
             </small>
           </td>
           <td>
-            <% menu.store_ids.each do |store| %>
-              <span class="badge badge-secondary"><%= @stores.find(store).unique_name %></span>
+            <% if menu.store_id %>
```

Comment:
we need to add `store` presence validation, there should not be a `menu` record without store assigned